R/tools.R

```diff
@@ -73,10 +73,16 @@ btw_tools <- function(...) {
     }
   )
 
-  tools_to_keep <- map_lgl(.btw_tools, is_tool_match, tools)
-  res <- .btw_tools[tools_to_keep]
+  tools_is_match <- map_lgl(.btw_tools, is_tool_match, tools)
+  res <- .btw_tools[tools_is_match]
```

Comment:
```suggestion
  tools_to_keep <- map_lgl(.btw_tools, is_tool_match, tools)
  res <- .btw_tools[tools_to_keep]
```

The old naming, perhaps🐛